web-server/src/utils/logFormatter.ts

```diff
@@ -0,0 +1,95 @@
+import { LogLevel } from '@/types/resources';
+
+import {
+  ParsedLog,
+  generalLogRegex,
+  httpLogRegex,
+  redisLogRegex,
+  postgresLogRegex,
+  dataSyncLogRegex
+} from '../constants/log-formatter';
+
+export const parseLogLine = (rawLogLine: string): ParsedLog | null => {
+  const generalLogMatch = rawLogLine.match(generalLogRegex);
+  if (generalLogMatch) {
+    const [_fullLog, timestamp, _unused, logLevel, message] = generalLogMatch;
```

Comment:
`_unused`
this looks off, name it correctly to what this variable stands for, and the underscore is enough to know that this variable is unused

Reply:
done. ,changing it with _pid